src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+.conteiner {
+  display: flex;
+  justify-content: center;
+  margin: 100px;
+}
+
+.arrow-minutes {
+  background-color: red;
+  width: 10px;
+  height: 150px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  margin-top: 50px;
+  margin-left: -11px;
+  transform-origin: bottom;
+  animation: time 3s linear alternate forwards;
```

Comment:
Why do you animate only 3 seconds?
You should animate continuously: seconds will be animated full round for 60 seconds, and minutes for 60 minutes (3600s)